packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -65,8 +66,17 @@ export async function beaconHandler(args: BeaconArgs & GlobalArgs): Promise<void
 
   if (ACTIVE_PRESET === PresetName.minimal) logger.info("ACTIVE_PRESET == minimal preset");
 
-  const db = new BeaconDb(config, await LevelDbController.create(options.db, {metrics: null, logger}));
-  logger.info("Connected to LevelDB database", {path: options.db.name});
+  let controller: LevelDbController | LmdbController;
+  if (options.db.type === "level") {
+    controller = await LevelDbController.create(options.db, {metrics: null, logger});
+    logger.info("Connected to LevelDB database", {path: options.db.name});
+  } else if (options.db.type === "lmdb") {
+    controller = await LmdbController.create(options.db, {metrics: null, logger});
+    logger.info("Connected to LMDB database", {path: options.db.name});
+  } else {
+    throw new Error(`Unsupported db type: ${options.db.type}`);
+  }
```

Comment:
That logic might go well as static method inside `BeaconDb` as `BeaconDB.init`. This will help us reuse it.